di_website/templates/blog/footnotes.html

```diff
@@ -0,0 +1,11 @@
+{% load string_utils wagtailcore_tags %}
+{% if page.footnotes_list %}
+    <div class="m-notes">
+      <h3 class="m-notes__title">Notes</h3>
+      <ul class="m-notes__list">
+        {% for item in page.footnotes_list %}
+          <li><sup><a id="{{ item.footnote_id }}" href="#{{ item.source_id }}">{{ forloop.counter }}</a></sup>{{ item.text | urlize }}</li>
```

Comment:
I don't think it's supposed to have the `<sup>` element here; only in the text ... looks strange here

![image](https://user-images.githubusercontent.com/5672438/69524111-68fa1980-0f76-11ea-93bd-dd0a0fc9b055.png)

Reply:
Copied the markup directly from Surge @edwinmp 
![image](https://user-images.githubusercontent.com/2836840/69544621-190f5880-0f5e-11ea-81ac-1c7d32f4840b.png)

http://development-initiatives.surge.sh/page-templates/12-blog-single

Reply:
Ah, in that case...